src/components/CanvasContainer/index.jsx

```diff
@@ -1,12 +1,70 @@
-import ChildCanvas from "../ChildCanvas";
+import { useState, useRef } from "react";
+
 import ParentCanvas from "../ParentCanvas";
+import CanvasUnit from "../CanvasUnit";
 
 const CanvasContainer = ({
   style,
   sketchTitle,
   onSketchTitleChange,
   elements,
 }) => {
+  const canvasRefs = {
+    headCanvas: useRef(null),
+    faceCanvas: useRef(null),
+    bodyCanvas: useRef(null),
+  };
+
+  const [headUnit, setHeadUnit] = useState({
+    width: 200,
+    height: 200,
+    y: 150,
+    x: 250,
+  });
+
+  const [faceUnit, setFaceUnit] = useState({
+    width: 150,
+    height: 90,
+    y: 250,
+    x: 275,
+  });
+
+  const [bodyUnit, setBodyUnit] = useState({
+    width: 200,
+    height: 200,
+    y: 325,
+    x: 250,
+  });
+
+  const mergeCanvasAndSave = () => {
+    const headCanvas = canvasRefs.headCanvas.current;
+    const faceCanvas = canvasRefs.faceCanvas.current;
+    const bodyCanvas = canvasRefs.bodyCanvas.current;
+
+    const mergedCanvas = document.createElement("canvas");
+    mergedCanvas.width = 700;
+    mergedCanvas.height = 700;
+
+    const mergedContext = mergedCanvas.getContext("2d");
+
+    mergedContext.fillStyle = "white";
+    mergedContext.fillRect(0, 0, mergedCanvas.width, mergedCanvas.height);
```

Comment:
```suggestion
    const mergedContext = mergedCanvas.getContext("2d");
    mergedContext.fillStyle = "white";
    mergedContext.fillRect(0, 0, mergedCanvas.width, mergedCanvas.height);
```
윗 부분과 통일하기 위해 여기도 개행을 없애는 것이 좋을 것 같아요!

Reply:
네! 그렇네요~ 반영하였습니다. 감사해요 😄